src/ducks/transactions/TransactionsPage.spec.jsx

```diff
@@ -80,7 +80,15 @@ describe('TransactionsPage', () => {
       date: '2018-04-30T12:00'
     })
     expect(instance.setState).toHaveBeenCalledWith({
-      currentMonth: '2018-03'
+      currentMonth: '2018-04'
+    })
+    instance.handleChangeTopmostTransaction({
+      account: { data: { type: 'CreditCard' } },
```

Comment:
maybe you can add test to not CreditCard account

Reply:
yep, thanks